asreview/data.py

```diff
@@ -466,6 +473,13 @@ def get(self, name):
         except KeyError:
             return self.df[name].values
 
+    @property
+    def reverse_index(self):
+        if len(self._reverse_index) != len(self):
+            self._reverse_index = {
+                self.df.index.values[i]: i for i in range(len(self))}
+        return self._reverse_index
+
```

Comment:
Why do you cache this function if it is single-use?

Reply:
Otherwise multiple calls to ASReviewData.record(by_index=False) will be very slow.